content/pt/docs/concepts/cluster-administration/kubelet-garbage-collection.md

```diff
@@ -0,0 +1,71 @@
+---
+reviewers:
+title: Configurando o Garbage Collection do kubelet
+content_template: templates/concept
+weight: 70
+---
+
+{{% capture overview %}}
+
+O Garbage collection(Coleta de lixo) é uma função útil do kubelet que limpa imagens e recipientes não utilizados. O kubelet executará o garbage collection para contêineres a cada minuto e para imagens a cada cinco minutos.
```

Comment:
I think that you could keep the word container because recipientes is a little bit strange in this context.

Reply:
I agree @devlware, it was a lack of attention, when you want to suggest use the feature suggestion, example:

```suggestion
O Garbage collection(Coleta de lixo) é uma função útil do kubelet que limpa imagens e contêineres não utilizados. O kubelet executará o garbage collection para contêineres a cada minuto e para imagens a cada cinco minutos.
```